config/feature-flags.php

```diff
@@ -36,4 +36,28 @@
         */
         'status_code' => 404,
     ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Enabling Time bombs for Features
+    |--------------------------------------------------------------------------
+    |
+    | This option controls whether an exception will be thrown if a feature
+    | has expired. See Martin Fowler's blog post on this:
+    | https://martinfowler.com/articles/feature-toggles.html#WorkingWithFeature-flaggedSystems
+    |
+    */
+    'enable_time_bombs' => false,
+
+    /*
+     |--------------------------------------------------------------------------
+     | Environments that will NOT trigger Time Bombs
+     |--------------------------------------------------------------------------
+     |
+     | This option controls which environment settings will prevent time bomb
+     | exceptions from being thrown. To trigger in all environments, leave
+     | the array as empty.
+     |
+     */
+     'time_bomb_environments' => ['production']
```

Comment:
I find the name `time_bomb_environments` a little counter-intuitive. Without reading the comment, I would have assumed that these are the environments in which the time bombs are _active_, not the other way around. 

Another thing is that my first instinct have would be to have the time bombs be deactivated in every environment and to explicitly activate them for certain environments. I think this would reduce the chance of accidentally activating it in production because of a typo or something similar.

Reply:
I have to agree here, time bomb feature should be enabled per environment and off by default - not something you would typically have in staging/qa